### PR TITLE
Have comments show up

### PR DESCRIPTION
**Pull request by AirwreckEye**

### CODE REVIEW

File: src/main/java/com/google/ehub/servlets/ItemPageServlet.java [hunk 1]
```diff
@@ -66,13 +65,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     long itemId;
+    
+    System.out.println("TEST:" + request.getParameter("itemId"));
```
Comment (BryanTriana) [inline src/main/java/com/google/ehub/servlets/ItemPageServlet.java:69]: I think you should remove this testing print statements from the PR since they shouldn't be part of master and the production code

File: src/main/java/com/google/ehub/servlets/ItemPageServlet.java [hunk 2]
```diff
@@ -66,13 +65,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
     try {
-      itemId = Long.parseLong(request.getParameter(CommentDataManager.ITEM_ID_PROPERTY_KEY));
+      itemId = Long.parseLong(request.getParameter("itemId"));
```
Comment (BryanTriana) [inline src/main/java/com/google/ehub/servlets/ItemPageServlet.java:71]: remember to use a constant variable for the parameter keys like this:
```private static final String ITEM_ID_PARAMETER_KEY = "itemId";```

File: src/main/webapp/item-page.html [hunk 3]
```diff
@@ -1,37 +1,33 @@
...
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+</head>
+
+<body onload="loadItemPage()">
+	<div class="container">
+		<div id="item-container"></div>
+		<footer>
+			<ul id="comment-container"></ul>
+			<form>
```
Comment (BryanTriana) [inline src/main/webapp/item-page.html:24]: Apply prettier formatter to fix indentation by two spaces instead of four.

File: src/main/webapp/itempagescript.js [hunk 4]
```diff
@@ -5,12 +5,10 @@ async function loadItemPage() {
   const itemId = getItemId();
   console.log(itemId);
   if (itemId != null) {
-    fetch(`/itempagedata?itemId=${itemId}`)
-      .then((response) => response.json())
-      .then((ItemPageData) => {
-        createSelectedItemCard(ItemPageData.item);
-        getItemPageComments(ItemPageData.comments);
-      });
+    fetch(`/itempagedata?itemId=${itemId}`).then((response) => response.json()).then((ItemPageData) => {
```
Comment (BryanTriana) [inline src/main/webapp/itempagescript.js:8]: The variable ```ItemPageData``` should be in camelcase like this: ```itemPageData```

File: src/main/webapp/itempagescript.js [hunk 5]
```diff
@@ -47,11 +41,13 @@ function getItemId() {
...
  */
 async function sendFormData() {
+  const comment = $('#comment').val();
   const itemId = getItemId();
-  const comment = document.getElementById('comment');
-  fetch(
-    `/itempagedata?=${itemId}`,
-    { method: 'post', body: JSON.stringify(comment) })
+  $.post('/itempagedata', { comment: comment, itemId: itemId }).done(function() {
+    window.location.reload();
```
Comment (BryanTriana) [inline src/main/webapp/itempagescript.js:47]: Does the page really need to reload to submit a new comment or can ItemPageServlet redirect to the page by itself?

File: src/main/webapp/itempagescript.js [hunk 6]
```diff
@@ -5,12 +5,10 @@ async function loadItemPage() {
   const itemId = getItemId();
   console.log(itemId);
```
Comment (BryanTriana) [inline src/main/webapp/itempagescript.js:6]: Remove the print statements after you get done with the testing.

File: src/main/webapp/itempagescript.js (hunk 5 above)
Comment (AirwreckEye) [reply src/main/webapp/itempagescript.js:47]: If I don't, it'll accept the same comment multiple times. The servlet code doesn't redirect it.

File: src/main/webapp/item-page.html [hunk 7]
```diff
@@ -1,37 +1,33 @@
...
-        <form>
-          <textarea name="comment"></textarea>
-          <br /><br />
-          <button onclick="sendFormData()">Submit</button>
-        </form>
-      </footer>
-    </div>
-  </body>
-</html>
+<head>
```
Comment (codewithjess) [inline src/main/webapp/item-page.html:4]: did you change this file, besides the formatting? if not, you can revert this file back to whatever master has

File: src/main/webapp/itempagescript.js [hunk 8]
```diff
@@ -24,11 +22,7 @@ async function createSelectedItemCard(entertainmentItem) {
   card.append(
     $('<img class="card-img-top" src="' + entertainmentItem.imageUrl + '">'));
   const cardBody = $('<div class="card-body"></div>');
-  cardBody.append(
-    $('<h5 class="card-title">' + entertainmentItem.title + '</h5>'));
```
Comment (codewithjess) [inline src/main/webapp/itempagescript.js:28]: Why did you remove the title?

Comment (AirwreckEye) [reply src/main/webapp/itempagescript.js:28]: This code is outdated and for some reason, my pushes aren't affecting the git
